src/docs/guides/healthchecks-and-restarts.md

```diff
@@ -16,10 +16,14 @@ To configure a healthcheck -
 
 ## Configure Healthcheck Port
```

Comment:
i think this section needs an example or a screenshot

Reply:
IMO a screenshot would be a tad overkill when all the user needs to do is set a service variable, I'll see what I can do to make that more straightforward.

Reply:
Okay you've convinced me, I'll see what I can do for a screenshot that doesn't take up half the page.

Reply:
doesn't have to be crazy, can even come back to it, it's minor